ts/packages/borsh/src/index.ts

```diff
@@ -8,8 +8,9 @@ import {
   u8,
   union,
 } from "buffer-layout";
-import { PublicKey } from "@solana/web3.js";
+import { Base58EncodedAddress } from "web3js-experimental";
 import BN from "bn.js";
+import { base58 } from "@metaplex-foundation/umi-serializers";
```

Comment:
Can include `bs58` package instead since `anchor` package also has it.

Reply:
I actually hope we can _all_ move away from `bs58` so that we can incorporate performance improvements like [this one](https://github.com/cryptocoinjs/base-x/pull/81) that aren't likely to ever make it in.

Reply:
That sounds great but it's also likely to cause duplicate code in people's bundled code since there is a lot of other packages that will still depend on `bs58` package.